pkg/server/local_observer_test.go

```diff
@@ -231,3 +231,64 @@ func TestLocalObserverServer_OnFlowDelivery(t *testing.T) {
 	// Only every second flow should have been received
 	assert.Equal(t, flowsReceived, numFlows/2)
 }
+
+func TestLocalObserverServer_OnGetFlows(t *testing.T) {
+	numFlows := 100
+	queueSize := 0
+	req := &observer.GetFlowsRequest{Number: uint64(100)}
+	flowsReceived := 0
+	fakeServer := &FakeGetFlowsServer{
+		OnSend: func(response *observer.GetFlowsResponse) error {
+			flowsReceived++
+			return nil
+		},
+		FakeGRPCServerStream: &FakeGRPCServerStream{
+			OnContext: func() context.Context {
+				return context.Background()
+			},
+		},
+	}
+
+	var key int
+	onGetFlows := func(ctx context.Context, req *observer.GetFlowsRequest) (context.Context, error) {
+		return context.WithValue(ctx, key, 10), nil
```

Comment:
See `golint`: `should not use basic type int as key in context.WithValue`

Introducing a `type ctxKey int` should be sufficient to satisfy the lint.

Reply:
Yep, this will need a type we can use for this consistently.